### DISCUSSION TITLE
Watched paths

### DISCUSSION THREAD

**Discussion by DeNelo** [mondeja/mkdocs-include-markdown-plugin#245] [Q&A] [answered]
When running `mkdocs -serve`, mkdocs adds a watch path for every included file. The log entry "Watching paths for changes" gets really long.
In my case, that is not necessary. Is there any way to avoid that?

**Comment by mondeja** [comment]
Moved to #246

**Comment by mondeja** [comment, marked as answer]
You need to report this to Mkdocs itself because there is not a way to silent these messages at plugin level.

**Reply by NFanoe** [reply]
Maybe make adding the files to watch list an option?
https://github.com/mondeja/mkdocs-include-markdown-plugin/blob/253cbc4a3c337706a594773a8811de9c10cc00c9/src/mkdocs_include_markdown_plugin/plugin.py#L73-L74

**Reply by mondeja** [reply]
The user is requesting about not to log messages on the console. I'm not sure how not to watch  files at all would be useful within this regard.

You can also skip `mkdocs build` and your console will be clean forever 😅
